### DISCUSSION TITLE
how do i configure from the command line the proformtreadmillip ?

### DISCUSSION THREAD

**Discussion by pssara** [cagnulein/qdomyos-zwift#1159] [General]
Hi there,
I have compiled the application on raspberry pi and i want to set the proformtreadmillip. How can i do that ?
many thanks,
Panagiotis

**Comment by cagnulein** [comment]
@pssara you have to start the UI interface. or do you want to run it headless? In this case i need to add a command line setting for this
Let me know

**Reply by pssara** [reply]
I cannot get the UI to start on raspberry. Can i copy somehow the configuration from my MAC ?

**Reply by cagnulein** [reply]
you can copy the configuration file, but it's better to use a setting for this. Why you can't see the UI? Are you running a raspberry with a screen? if not, why don't you use VNC?

**Reply by pssara** [reply]
Hi there. Yes I have a screen and input devices, desktop is running, but the application just exits. I will try the VNC but maybe you can tell where can I find the config file ?

**Reply by cagnulein** [reply]
if it exits it's because you need to install the qml module
run a 

apt install *qml*

the config file is somewhere in the /home/username/.config/qt/qz  (something similar, i'm not on a PC right now)
